test/fastboot-test.js

```diff
@@ -101,16 +101,26 @@ describe("FastBoot", function() {
       });
   });
 
-  it("can forcefully destroy the app instance using destroyAppInstanceInMs", function() {
+  it("can forcefully destroy the app instance using destroyAppInstanceInMs", function(done) {
     var fastboot = new FastBoot({
       distPath: fixture('basic-app')
     });
 
-    return fastboot.visit('/', {
-      destroyAppInstanceInMs: 5
-    })
+    // delaying `visitRoute` to forcefully destroy app instance
+    let originalVisitRoute = fastboot._app.visitRoute;
```

Comment:
Instead of mutating the private API in the test to force a condition, please create a fixture that does a `setTimeout` in the model hook which implicitly will delay the `visit` resolution.

It seems `basic-app` does not do this currently.